docs/profile/vc.rst

```diff
@@ -199,17 +201,23 @@ Example
   const storedVcDoc = await vc.getVc('0x2a838a6961be98f6a182f375bb9158848ee9760ca97a379939ccdf03fc442a23');
   const otherStoredVcDoc = await vc.getVc('vc:evan:testcore:0x2a838a6961be98f6a182f375bb9158848ee9760ca97a379939ccdf03fc442a23');
 
```

Comment:
Are these the same docs in your example?
0x2a838a6961be98f6a182f375bb9158848ee9760ca97a379939ccdf03fc442a23
vc:evan:testcore:0x2a838a6961be98f6a182f375bb9158848ee9760ca97a379939ccdf03fc442a23

Better use different docs to not confuse anyone